src/Microsoft.Diagnostics.Runtime/src/Utilities/PEImage/PEImage.cs

```diff
@@ -383,13 +442,25 @@ private ResourceEntry CreateResourceRoot()
             return t;
         }
 
+        private unsafe void Write<T>(T data) where T : unmanaged
+        {
+            int size = Unsafe.SizeOf<T>();
+            byte[] buffer = new byte[size];
+            Span<byte> span = new Span<byte>(&data, size);
```

Comment:
Would wrapping `buffer` into a Span, and using `CopyTo` generate code using intrinsics here? Is there any value in doing so?

Reply:
> I put some comments into the implementation but at its core I don't like the approach of making a copy of the PE Image in memory and editing it for a few reasons:

@leculver, I will work on the translation table approach.